src/drivers/PortFolder/clean.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off

```

Comment:
Please include a summary of this command.

Reply:
Removed. It wasn't needed.